src/common.mjs

```diff
@@ -111,6 +111,16 @@ export function defaultTrackColors(trackType){
   }
 }
 
+/* Function to determine if reads are added, where tracks is an object */
```

Comment:
Tracks isn't just any object, right? It is meant to be an object full of track objects (which is a type we've defined).

And by "reads are added", we mean that we're checking to see if any of the tracks is a read track. I'm not sure "added" is the right word here; we're checking to see if they are present.